README.md

```diff
@@ -1,5 +1,7 @@
 # General Putnam Motel Diner API
 
+Adapted from Alex Fiedler 
```

Comment:
please add link to Alex's LinkedIn page with the exercise - the link is in the document which is on the google drive.